lab-adan/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "lab-adan",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "test": "mocha"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
You need to change the value here to be the name of our testing tool (in our case, we're using mocha).